### PR TITLE
Cargo.toml: fix missing fields for publishing v0.1.0

### PR DESCRIPTION
**Pull request by andelf**

### CODE REVIEW

**Comment by andelf** [comment]
@messense @MnO2 Is the `exclude` field alright? for excluding entire `priv/`.

**Comment by messense** [comment]
Run `cargo package` and verify that in `target/package`.

File: Cargo.toml [hunk 1]
```diff
@@ -2,21 +2,29 @@
...
+authors = ["andelf <andelf@gmail.com>",
+           "Paul Meng <me@paulme.ng>",
+           "messense <messense@icloud.com>"]
 readme = "README.md"
-keywords = ["trie", "darts"]
-categories = ["encoding"]
+keywords = ["trie", "darts", "string", "search", "text"]
+categories = ["data-structures", "text-processing"]
 edition = "2018"
+exclude = ["/benches", "/priv", "/.travis.yml"]
```
Comment (messense) [inline Cargo.toml]: `exclude = ["/benches/**", "/priv/**", "/.travis.yml"]` should work.

https://github.com/rust-lang/cargo/issues/4268

**Comment by andelf** [comment]
```
> cargo package
warning: Pattern matching for Cargo's include/exclude fields has changed and file `benches/darts_benchmark.rs` is now excluded.
See <https://github.com/rust-lang/cargo/issues/4268> for more information.
warning: Pattern matching for Cargo's include/exclude fields has changed and file `priv/dict.big.bincode` is now excluded.
See <https://github.com/rust-lang/cargo/issues/4268> for more information.
warning: Pattern matching for Cargo's include/exclude fields has changed and file `priv/dict.txt.big` is now excluded.
See <https://github.com/rust-lang/cargo/issues/4268> for more information.
warning: Pattern matching for Cargo's include/exclude fields has changed and file `priv/weicheng.txt` is now excluded.
See <https://github.com/rust-lang/cargo/issues/4268> for more information.
   Packaging darts v0.1.0 (/Users/***/github/rust-darts)
   Verifying darts v0.1.0 (/Users/***/github/rust-darts)
    Updating crates.io index
   Compiling darts v0.1.0 (/Users/***/github/rust-darts/target/package/darts-0.1.0)
    Finished dev [unoptimized + debuginfo] target(s) in 2.36s
```

```
darts-0.1.0.crate
├── .cargo_vcs_info.json
├── .editorconfig
├── .gitignore
├── Cargo.toml
├── Cargo.toml.orig
├── LICENSE
├── README.md
├── rustfmt.toml
└── src
    ├── lib.rs
    └── searcher.rs
```

**Review by messense — APPROVED** [review]
LGTM

**Comment by andelf** [comment]
Published as 0.1.0. 🎉

**Pull request merged**